pages.pl/osx/w.md

```diff
@@ -0,0 +1,17 @@
+# w
+
+> Pokazuje kto jest zalogowany i co aktualnie robi.
+> Wyświetla login, TTY, zdalny host, czas zalogowania, czas bezczynności i aktualny proces.
+> Więcej informacji: <https://manned.org/w>.
```

Comment:
The osx version for `w` doesn't have long arguments, only 1 letter.
```suggestion
> Więcej informacji: <https://ss64.com/osx/w.html>.
```
Please add this link to the main page too.